Realm/RLMSyncUser.h

```diff
@@ -77,6 +80,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  The current state of the user.
+
+ @see `RLMSyncUserState`
```

Comment:
Why this `@see`? It just points to the type this is documenting. `@see` blocks are used to refer to _other_ types.